index.html

```diff
@@ -0,0 +1,499 @@
+          <p class="related-titles-font"><a class="most-popular-links" href="https://www.nytimes.com/2020/09/25/opinion/debate-trump-biden-lie.html">Opinion: How to Debate Someone Who Lies</a></p>
+          <p class="related-titles-font"><a class="most-popular-links" href="https://www.nytimes.com/2020/09/25/world/asia/trump-united-states.html">‘I Feel Sorry for Americans’: A Baffled World Watches the U.S.</a></p>
+          <p class="related-titles-font"><a class="most-popular-links" href="https://www.nytimes.com/2020/09/25/us/politics/amy-coney-barrett-supreme-court.html">Trump Selects Amy Coney Barrett to Fill Ginsburg’s Seat on the Supreme Court</a></p>
+          <p class="related-titles-font"><a class="most-popular-links" href="https://www.nytimes.com/2020/09/26/opinion/amy-coney-barrett-supreme-court.html">Opinion: The Meaning of Amy Coney Barrett</a></p>
+          <p class="related-titles-font"><a class="most-popular-links" href="https://www.nytimes.com/2020/09/25/opinion/trump-election-supreme-court.html">Opinion: Will Trump’s Presidency Ever End?</a></p>
+          <p class="related-titles-font"><a class="most-popular-links" href="https://www.nytimes.com/2020/09/25/obituaries/cathy-phanavong-dead-Covid.html">Catharine Phanavong, Expert Cook and Dinner Party Enthusiast, Dies at 39</a></p>
+          <p class="related-titles-font"><a class="most-popular-links" href="https://www.nytimes.com/2020/09/25/us/politics/trump-military-election.html">At Pentagon, Fears Grow That Trump Will Pull Military Into Election Unrest</a></p>
+          <p class="related-titles-font"><a class="most-popular-links" href="https://www.nytimes.com/2020/09/24/style/first-language-at-home-bilingual.html">Mom, Can’t We Speak English Here?</a></p>
+        </div>
+      </section>
+      <!-- End of Most Popular Side Section -->
+    </aside>
+  </main>
+  <!-- Footer Links -->
+  <footer class="bottom-width">
+    <a href="https://www.nytimes.com/"><img src="assets/images/logo-the-nyt.png" alt="the new york times logo"></a>
+    <p><small><a href="#">Go to Home Page >></a></small></p>
+    <div class="footer-link-wrapper d-flex">
+      <article>
+        <h4>NEWS</h4>
+        <ul>
+          <li><a href="#">Home Page</a></li>
+          <li><a href="#">World</a></li>
+          <li><a href="#">U.S.</a></li>
+          <li><a href="#">Politics</a></li>
+          <li><a href="#">Election 2020</a></li>
+          <li><a href="#">New York</a></li>
+          <li><a href="#">Business</a></li>
+          <li><a href="#">Tech</a></li>
+          <li><a href="#">Science</a></li>
+          <li><a href="#">Sports</a></li>
+          <li><a href="#">Obituaries</a></li>
+          <li><a href="#">Today's Paper</a></li>
+          <li><a href="#">Corrections</a></li>
+        </ul>
+      </article>
```

Comment:
This is a wrong semantic usage, use `nav` instead as this part is not an article but navigation links.

Update the same for the rest of the links.